client/verta/tests/unit_tests/internal_utils/test_kafka_utils.py

```diff
@@ -2,81 +2,80 @@
 
 """Unit tests for Kafka utilities."""
 
-from typing import Any, Dict, List
+from typing import Any, Dict
 
-import pytest
+from hypothesis import given, HealthCheck, settings, strategies as st
 
 from verta._internal_utils import kafka
 
 
-@pytest.fixture
-def mock_kafka_configs_response() -> Dict[str, Any]:
+@st.composite
+def mock_kafka_configs_response(draw) -> Dict[str, Any]:
     """
     Provide mocked API result from `api/v1/uac-proxy/system_admin/listKafkaConfiguration`
     with a single Kafka configuration.
     """
+    recursive_json = st.recursive(
+        st.none() | st.booleans() | st.integers() | st.text(),
+        lambda children: st.lists(children) | st.dictionaries(st.text(), children),
+        max_leaves=5,
+    )
```

Comment:
We have something almost identical to this as [`tests.strategies.json_strategy`](https://github.com/VertaAI/modeldb/blob/08681e3541f15a67000e21e05aa053a754fa26e7/client/verta/tests/strategies.py#L25-L30). Could we possibly reuse that?